src/ripple/overlay/impl/OverlayImpl.h

```diff
@@ -456,6 +456,62 @@ class OverlayImpl : public Overlay
 
     void
     sendEndpoints();
+
+private:
+
+    struct TrafficGauges{
```

Comment:
Micro-nit: this is, technically, a `TrafficGauge` not `TrafficGauges`. I don't think this is a particularly big deal, but I think the change is basically a search-and-replace so it shouldn't be a big deal.

Reply:
Isn't it 4 gauges (bytes/messages in/out) in one structure?

Reply:
@nbougalis Since there are multiple gauges in the struct, shouldn't it remain TrafficGauges?